service-loadbalancer/service_loadbalancer.go

```diff
@@ -25,6 +25,7 @@ import (
 	"os"
```

Comment:
can we just delete service_loadbalancer alltogether? I'd rather have a placeholder README in serviceloadbalancer/ that says how anyone who wants serviceloadbalancer functionality can spin up this Ingress controller with --use-ingress=false